Plutarch/Api/V1.hs

```diff
@@ -159,6 +148,8 @@ newtype PScriptContext (s :: S)
                ]
           )
       )
+  deriving stock (GHC.Generic)
+  deriving anyclass (Generic)
```

Comment:
Since people are used to seeing Generic = GHC.Generic, might be a good idea to disambiguate harder, and use it as SOP.Generic